CIP-0006/CIP-0006.md

```diff
@@ -0,0 +1,157 @@
+|  `name` | A name for the pool | 50 Characters Maximum |
+
+Many additional attributes can be envisioned for future wallets, pool explorers, and information aggregators.  The proposal below outlines an initial strategy for capturing this extended metadata.
+
+## Specification
+
+### On Chain referenced (main) metadata file
+We define two more fields for the on chain referenced metadata file that references another json file on a url with the extended metadata.  The proposed metadata is as follows:
+
+| key           | Value                                | Rules  |
+| ---           | ---                                  | ---  |
+|  `ticker`       | Pool ticker.  uppercase              | 5 Characters Maximum, Uppercase letters and numbers  |
+|  `description` | Pool Description.  Text that describes the pool | 50 Characters Maximum |
+|  `homepage` | A website URL for the pool| 64 Characters Maximum, must be a valid URL |
+|  `name` | A name for the pool | 50 Characters Maximum |
+| `extDataUrl` | A URL for extended metadata | optional, 128 Characters Maximum, must be a valid URL |
+| `extHashUrl` | A URL with the extended metadata hash | optional, 128 Characters Maximum, must be a valid URL |
+| `extVkey` | the public Key for verification | optional, 68 Characters |
+
+In order to include the additional ext Field data, we suggest increasing the maximum size of the main metadata file from currently 512 to 1024 bytes.
+
+### Extended Metadata - flexible but validable
+
+In difference to the main metadata, the extended metadata should be updateable without having to use the cold key of the pool and without having to perform an on-chain transaction. The consumer of these data should still have the possibility to verify the authenticity of the data.
+
+The operator notes all his additional pool information in the extended metadata (`extData.json`).
+
+We propose the pool operator generate a new public-secret key-pair (`extData.skey` and `extData.vkey`)
+
+```shell
+cardano-cli shelley node key-gen 
+  --cold-verification-key-file extData.vkey 
+  --cold-signing-key-file extData.skey 
+  --operational-certificate-issue-counter-file extData.counter
+```
+
+Then a new (not available yet) `cardano-cli` command generate the signed hash (`extData.sign`) .
+
+```shell
+cardano-cli shelley stake-pool rawdata-hash
+  --raw-metadata-file extData.json
+  --signing-key-file extData.skey
+  --out-file extData.sign
+```
+
+The operator now:
+
+- has the `extData.json` and `extData.sign` files
+- will publish them at some https:// URL (probably same host as the main metadata)
+- use the `extData.vkey` string and the two extend file URLs to re-register the main metadata
+
+This re-registration of the main metadata file with the `extData.vkey` and the two URLs is only necessary once. Afterwards, the operator can update his extended metadata at any time, generate the new signature and put both files online.
```

Comment:
I think I understand, but the text is not clear.

Make clear which fields in the main metadata file these things correspond to. Say that you need the URL of the extended metadata json file, and signature file, that these are to fill in the `extDataUrl` and `extSigUrl` fields.

And what format exactly is the `extVkey`? Bech32 I presume? What prefix?

Reply:
@gufmar - can we get this addressed to have it merged next meeting?

Reply:
How about `poolmd_vk` for the bech32 prefix?